### DISCUSSION TITLE
Help Identifying Visual Studio Theme? [Image Attached]

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/vscode-discussions#952] [Extension Development QnA] [answered]
Hi everyone! 👋 I stumbled upon this Visual Studio screenshot, and I'm loving the theme. Can anyone help me identify it? Here's the image: https://imgur.com/a/S8GipBy . Thanks in advance for your help! 🙏

**Comment by olguzzar** [comment, marked as answer]
It looks like it could be one of the [GitHub Dark themes](https://marketplace.visualstudio.com/items?itemName=GitHub.github-vscode-theme). The screenshot also uses the[ vscode-icons extension](https://marketplace.visualstudio.com/items?itemName=vscode-icons-team.vscode-icons) to get that custom file icon!

This forum is specifically for people developing extensions, so I'm going to close this discussion.

**Comment by behruzerkinov** [comment]
![Screenshot 2024-10-09 204316](https://github.com/user-attachments/assets/e2e18485-5ec8-4bde-a573-208f8bd6ba67)

Which theme is that Mosh using in his courses?

**Reply by gjsjohnmurray** [reply]
@behruzerkinov:

@olguzzar previously wrote:

> This forum is specifically for people developing extensions, so I'm going to close this discussion.

Please see pinned thread https://github.com/microsoft/vscode-discussions/discussions/5 for other pointers.
